stable/prometheus-adapter/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus-adapter
-version: v0.1.2
+version: v0.1.3
```

Comment:
Bump version to 0.2.0 since it is a new fwature

Reply:
Thanks, made the changes.